doc/source/whatsnew/v1.1.0.rst

```diff
@@ -426,6 +426,7 @@ Performance improvements
   :meth:`DataFrame.sparse.from_spmatrix` constructor (:issue:`32821`,
   :issue:`32825`,  :issue:`32826`, :issue:`32856`, :issue:`32858`).
 - Performance improvement in reductions (sum, prod, min, max) for nullable (integer and boolean) dtypes (:issue:`30982`, :issue:`33261`, :issue:`33442`).
+- Performance improvement in :meth:`DataFrameGroupBy.func` when aggregating categorical data (:issue:`32976`)
```

Comment:
```suggestion
- Performance improvement in :meth:`DataFrame.groupby` when aggregating categorical data (:issue:`32976`)
```

Reply:
can you update as @TomAugspurger indicated (and don't just resolve)

Reply:
my bad, meant to amend that last time around